dra-evolution/testdata/pod-one-container-one-gpu-one-vf.yaml

```diff
@@ -57,23 +57,31 @@ spec:
   constraints:
   - matchAttribute: dra.k8s.io/pcie_root
   requests:
-  - deviceClassName: foozer.example.com
-  - deviceClassName: sriov-nic-example.org
+  - name: gpu
+    deviceClassName: foozer.example.com
```

Comment:
We decided to make request names mandatory. Not all examples have been updated, so I am doing it here now because I am touching it.